packages/import-utils/src/create-sandbox/templates.ts

```diff
@@ -199,6 +191,9 @@ export function getTemplate(
     "ts-node",
     "@tensorflow/tfjs-node",
     "webpack-dev-server",
+    "@sveltech/routify",
+    "vite",
```

Comment:
These ones should still be on top, to prevent eg. that `svelte` returns early because of the check o line 183. Other than that it looks good!

Reply:
That makes a lot of sense 

Will change and merge

Reply:
Fixed